lib/src/spec/validationEngine.spec.ts

```diff
@@ -6,168 +6,198 @@ import { FieldValidationResult } from '../entities';
 
 //TODO: Implement Issue #20
 describe('lcFormValidation', () => {
-    it('should return isFormPristine true after initialization', () => {
-        // Arrange
-        const formValidationBase : ValidationEngine = new ValidationEngine();
+  it('should return isFormPristine true after initialization', () => {
+    // Arrange
+    const formValidationBase : ValidationEngine = new ValidationEngine();
 
-        // Act
-        formValidationBase.initialize([]);
+    // Act
+    formValidationBase.initialize([]);
 
-        // Assert
-        expect(formValidationBase.isFormPristine()).to.be.true;
-    });
-
-    it('should return isFormPristine false after initialization and set value field', (done) => {
-        // Arrange
-        const formValidationBase : ValidationEngine = new ValidationEngine();
-        const viewModel = [{formFieldName: 'nameId', vmFieldName: 'name'}];
-
-        // Act
-        formValidationBase.initialize(viewModel);
-
-        formValidationBase
-          .triggerFieldValidation(viewModel, 'nameId', 'newContent')
-          .then((errors) => {
-            // Assert
-            expect(formValidationBase.isFormPristine()).to.be.false;
-            done();
-          });
-    });
+    // Assert
+    expect(formValidationBase.isFormPristine()).to.be.true;
+  });
 
+  it('should return isFormPristine false after initialization and set value field', (done) => {
+    // Arrange
+    const formValidationBase : ValidationEngine = new ValidationEngine();
+    const viewModel = [{formFieldName: 'nameId', vmFieldName: 'name'}];
 
-    it('should return isFormDirty false after initiazalition', () => {
-        // Arrange
-        const formValidationBase : ValidationEngine = new ValidationEngine();
-
-        // Act
-        formValidationBase.initialize([]);
+    // Act
+    formValidationBase.initialize(viewModel);
 
+    formValidationBase
+      .triggerFieldValidation(viewModel, 'nameId', 'newContent')
+      .then((errors) => {
         // Assert
-        expect(formValidationBase.isFormDirty()).to.be.false;
-    });
+        expect(formValidationBase.isFormPristine()).to.be.false;
+        done();
+      });
+  });
+
 
-    it('should return isFormDirty true after initialization and set value field',(done) => {
-        // Arrange
-        const formValidationBase : ValidationEngine = new ValidationEngine();
-        const viewModel = [{formFieldName: 'nameId', vmFieldName: 'name'}];
+  it('should return isFormDirty false after initiazalition', () => {
```

Comment:
Look at all the test suite, I think that variables are not consistent (let / const) declare.